taichi/ir/statements.h

```diff
@@ -912,15 +912,19 @@ class BitExtractStmt : public Stmt {
  */
 class GetRootStmt : public Stmt {
  public:
-  GetRootStmt() {
+  SNode *snode;
```

Comment:
nit: How about we call this `root`, so that it's more easy to be distinguished from the ctor parameter name.

Another personal note: it's better to privatize member variables, and provide a getter to access that (i.e., Python's `@property`). However, I know that most of Taichi's codebase is not following this style, so it's fine either way.